packages/asc-ui/src/components/TopTaskLink/TopTaskLinkStyle.ts

```diff
@@ -20,7 +20,7 @@ export const TopTaskTitle = styled(Paragraph)`
 `
 
 export default styled.a<{
-  onDarkBackground?: boolean
+  hasDarkBackground?: boolean
```

Comment:
I don't think we can simply rename this prop as we need to still support the deprecated variant.

Reply:
Good catch.

Fixed all improvements 👍